api-catalog-services/src/test/java/org/zowe/apiml/apicatalog/instance/InstanceRefreshServiceTest.java

```diff
@@ -37,175 +32,173 @@
 import static org.zowe.apiml.constants.EurekaMetadataDefinition.CATALOG_ID;
 import static org.mockito.Mockito.any;
 import static org.mockito.Mockito.anyString;
+import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.never;
 import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
-@ExtendWith(MockitoExtension.class)
-@MockitoSettings(strictness = Strictness.LENIENT)
+@Nested
```

Comment:
I think @Nested annotation is not necessary at the beginning of the class

Reply:
Thanks for catching it